packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -207,26 +220,38 @@ export const styles = css`
         css`
             .control {
                 background-color: rgba(${borderRgbPartialColor}, 0.1);
-                border-color: transparent;
             }
 
             .control:hover {
                 background-color: transparent;
-                border-color: ${borderHoverColor};
+                background-image: linear-gradient(
+                    rgba(${borderRgbPartialColor}, 0.1),
+                    rgba(${borderRgbPartialColor}, 0.1)
+                );
+                background-size: calc(100% - 4px) calc(100% - 4px);
+                background-repeat: no-repeat;
+                background-position: center;
```

Comment:
These only apply when a background-image is set, so we can just move them up to the `.control ()` selector and avoid duplication. Whenever an image is set these are always the same